src/components/CouponDetails/constants.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable import/prefer-default-export */
+
+export const COUPON_FILTER_TYPES = {
+  unassigned: 'unassigned',
+  unredeemed: 'unredeemed',
+  partiallyRedeemed: 'partially-redeemed',
+  redeemed: 'redeemed',
+};
+
+export const COUPON_FILTER_OPTIONS = {
+  unassigned: 'Unassigned',
+  unredeemed: 'Unredeemed',
+  partiallyRedeemed: 'Partially redeemed',
+  redeemed: 'Redeemed',
+};
+
+export const ACTION_LABELS = {
+  remind: 'Remind',
+  assign: 'Assign',
+  revoke: 'Revoke',
+};
+
+export const ACTION_TYPES = {
```

Comment:
Not sure if this would be more annoying or not, but why not do something like:
```
export const ACTIONS = {
  remind: {'label': 'Remind', 'value': 'remind'},
  revoke:
  [...]
}
```
Then they'd be referenced by ACTIONS.revoke.label and ACTIONS.revoke.action
The plus side: only have to remember to add the action to one spot.
Downside: Requires one further level of accessor?
(This goes for `COUPON_FILTER_*` too)

Reply:
Yeah, it got built this way because I did it one by one, but I like that.